glide.yaml

```diff
@@ -90,9 +54,5 @@ import:
   version: 045497edb6234273d67dbc25da3f2ddbc4c4cacf
 - package: google.golang.org/api
   version: c858ef4400610cbfd097ffc5f5c6e4a1a51eac86
-- package: google.golang.org/appengine
-  version: 12d5545dc1cfa6047a286d5e853841b6471f4c19
-- package: google.golang.org/cloud
```

Comment:
We depend pretty heavily on this API, so I'm a little hesitant to unpin it.  Thoughts?

Reply:
We don't directly consume these APIs. We only use the `google.golang.org/api` one; they're transient deps. I'm relying on glide doing the right thing here. They also don't change in glide.lock.